mongo_connector/oplog_manager.py

```diff
@@ -608,9 +608,8 @@ def get_oplog_cursor(self, timestamp=None):
             query['ts'] = {'$gte': timestamp}
             cursor = self.oplog.find(
                 query,
-                cursor_type=CursorType.TAILABLE_AWAIT)
-            # Applying 8 as the mask to the cursor enables OplogReplay
-            cursor.add_option(8)
+                cursor_type=CursorType.TAILABLE_AWAIT,
+                oplog_replay=True)
```

Comment:
This option was added in PyMongo 3.0, but mongo-connector requires PyMongo >= 2.9, so it's possible this option won't be supported.